daffodil-schematron/src/test/scala/org/apache/daffodil/validation/schematron/EmbeddedTesting.scala

```diff
@@ -52,8 +54,7 @@ trait EmbeddedTesting {
       val bos = new ByteArrayOutputStream()
       val r1 = dp.parse(
         new InputSourceDataInputStream(new ByteArrayInputStream(bytes)),
-        new XMLTextInfosetOutputter(bos, true))
-
+        new XMLTextInfosetOutputter(bos, true, XMLOutputStyle.Standard))
```

Comment:
Likewise, why even pass XMLOutputStyle.Standard here when it's the default?  I also like pretty = true better (self-documenting) than true by itself.